jatin/postgres_assignment/src/controller/classcontroller.ts

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable no-console */
+export { };
+
+const database = require('../services/classservices.ts');
+
+async function addClass(ctx: any) {
+  const obj = ctx.request.body;
+  if (obj.classID == null || obj.room == null || obj.subjectID == null) {
```

Comment:
use joi or any other npm package to validate your data

Reply:
yes, added